backend/pick-git/src/main/java/com/woowacourse/pickgit/authentication/presentation/interceptor/IgnoreAuthenticationInterceptor.java

```diff
@@ -26,15 +26,11 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         }
 
         String authentication = AuthorizationExtractor.extract(request);
-        request.setAttribute("authentication", getAppropriateAuthentication(authentication));
-        return true;
-    }
-
-    private String getAppropriateAuthentication(String authentication) {
-        if (!oAuthService.validateToken(authentication)) {
-            return null;
+        if (authentication != null && !oAuthService.validateToken(authentication)) {
```

Comment:
```suggestion
        if (Objects.nonNull(authentication) && !oAuthService.validateToken(authentication)) {
```

프로덕션 코드에서 null을 사용하는건 좋지 않아요. 눈가리고 아웅일수도 있는 리팩터링이지만 가속성이나 의미상으로 충분히 의미가 있습니다.

Reply:
가속성..? 👀